api/docker-entrypoint.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+info()
+{
+    echo "Info: $*"
+}
+
+warn()
+{
+    echo "Warning: $*"
+}
+
+fatal()
+{
+    echo "Fatal: $*"
+}
+
+get_couchdb_url()
+{
+  echo "http://$COUCHDB_USER:$COUCHDB_PASSWORD@$COUCHDB_SERVICE_NAME:5985"
+}
+
+wait_for_couchdb()
+{
+  local n=60
+
+  while [ "$n" -gt 0 ]; do
+    curl "`get_couchdb_url`" &>/dev/null
+    [ "$?" -eq 0 ] && return 0
+    sleep 3
```

Comment:
```suggestion
    sleep 1
```
I'm kind of an impatient person, but even still, as long as a computer is doing the checking, why not check more frequently?